test/json/test-core.json.d.ts

```diff
@@ -0,0 +1,21 @@
+export type TestMessage = {
+  /** The MF2 syntax source. */
+  src: string;
+  /** The locale to use for formatting. Defaults to 'en-US'. */
+  locale?: string;
+  /** Parameters to pass in to the formatter for resolving external variables. */
+  params?: Record<string, string | number | null | undefined>;
```

Comment:
include date type here?

Reply:
Nope, because this is JSON data which doesn't inherently support dates.